phpstan.neon

```diff
@@ -0,0 +1,13 @@
+includes:
+  - phpstan-baseline.neon
+parameters:
+  level: max
+  strictRules:
+    allRules: false
+    strictCalls: true
+    requireParentConstructorCall: true
+  inferPrivatePropertyTypeFromConstructor: true
+  treatPhpDocTypesAsCertain: false
+  paths:
+    - packages/typo3-docs-theme/src
+  tmpDir: .build/phpstan/
```

Comment:
What is the benefit of defining `tmpDir` explicitly? Default is `/tmp/phpstan`.
Using the default one: there is no need to introduce a new folder `.build`

Reply:
@jaapio I copied that from the guides. is there an advantage to this or should we use the default as suggested by @brotkrueml ?

Reply:
In DDEV the phpstan cache does not get stored in my file system then. Not sure if this is an advantage or disadvantage?

Reply:
the phpstan cache will speed up the analysis, if we configure this correctly it will work in ddev and docker, if you have php natively installed you will not have an advantage rather than that the cache will not be cleared when your system cleans the /tmp dir. 

I think it's good to configure this.

Reply:
would it make sense to call the directory .cache instead of .build to make sure it can and should be cleared if there are problems with cache?

Reply:
That would be much better, as the purpose of this folder is now clear. I suggest to use `.cache/phpstan`, maybe we have other dev caches later.

Reply:
I think, we could also move the cs-fixer cache there and maybe the phpunit cache?